atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -670,8 +672,19 @@ public void readsFromThoroughlySweptTableShouldFailWhenLocksAreInvalid() {
 
         tasks.add(Pair.of("getRowsColumnRange(TableReference, Iterable<byte[]>, ColumnRangeSelection, int)",
                 (t, heldLocks) -> {
-                    t.getRowsColumnRange(TABLE_SWEPT_THOROUGH, Collections.singleton(PtBytes.toBytes("row1")),
-                            new ColumnRangeSelection(null, null), batchHint);
+                    Iterators.getLast(
```

Comment:
why `getLast`?

Reply:
I want to drain the iterator to verify that we've actually gone through with the operation, otherwise we might have done nothing if the iterator was sufficiently lazy - this mimics what the other `getRowsColumnRange` test here does